docs/reference/commandline/build.md

```diff
@@ -462,6 +462,12 @@ more `--add-host` flags. This example adds a static address for a host named
 $ docker build --add-host docker:10.180.0.1 .
 ```
 
+For IPv6 addresses, the equivalent `host=ip` form may be clearer, for example:
```

Comment:
Related to my other comment; we can use the new format as default for the example, and make a mention of the old format.

(also /cc @dvdksn for suggestions)

Reply:
Ok, will do.